tests/forte/processors/predictor_test.py

```diff
@@ -0,0 +1,104 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+from ft.onto.base_ontology import Sentence, Token, EntityMention
+from forte.pipeline import Pipeline
+from forte.train_preprocessor import TrainPreprocessor
+from forte.data.readers.conll03_reader import CoNLL03Reader
+from forte.data.data_pack import DataPack
+from forte.data.converter import Converter
+from forte.data.extractor.attribute_extractor import AttributeExtractor
+from forte.data.extractor.seqtagging_extractor import BioSeqTaggingExtractor
+from forte.data.batchers import FixedSizeDataPackBatcherWithExtractor
+from forte.processors.base.batch_processor import Predictor
+
+
+class ProcessingBatcher(unittest.TestCase):
```

Comment:
Clas name should follow test class convention

Reply:
Got it.